Makefile

```
@@ -68,7 +68,7 @@ available-node = \
 	elif [ -x `command -v node` ] && [ -e `command -v node` ] && [ `command -v node` ]; then \
 		`command -v node` $(1); \
 	else \
-		echo "No available node, cannot run \"node $(1)\""; \
+		printf "No available node, cannot run \"node $(1)\"\n"; \
```

I'm not sure this is really necessary, `echo` with no arguments seems to be well supported. Also, shouldn't we replace the variable with `%s` if we're using `printf`?

```suggestion
		printf "No available node, cannot run \"node %s\"\n" "$(1)"; \
```

---

It seems to work without replacing it with a `%s`.